src/grocker/resources/docker/compiler-image/provision.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/sh
 set -xe
 
-apt install -qy ${SYSTEM_DEPENDENCIES:=}
-install --mode=0555 --owner=grocker /tmp/grocker/compile.py /home/grocker/compile.py
-install --mode=0777 --owner=grocker -d /home/grocker/packages
+if which apt; then
+    apt install -qy ${SYSTEM_DEPENDENCIES:=}
+
+    install --mode=0555 --owner=grocker /tmp/grocker/compile.py /home/grocker/compile.py
```

Comment:
Those two lines are common between Debian and Alpine, so they should be outside of this conditional block